src/status_im/subs.cljs

```diff
@@ -648,6 +649,12 @@
  (fn [multiaccount]
    (fleet/current-fleet-sub multiaccount)))
 
+(re-frame/reg-sub
+ :opensea-enabled?
+ :<- [:multiaccount]
+ (fn [{:keys [opensea-enabled?]}]
+   (or opensea-enabled? false)))
```

Comment:
why `or` here ?

Reply:
It's possible that this field is not returned. `false` makes it explicit. The other option is to do `(bool opensea-enabled?)`